CppCoreGuidelines.md

```diff
@@ -11551,16 +11552,16 @@ We also include lossy arithmetic casts, such as from a negative floating point t
     unsigned u = 0;
 
     u = d;                          // BAD
-    u = narrow_cast<unsigned>(d);   // OK (you asked for it): u becomes 0
+    u = narrow_cast<unsigned>(d);   // OK (you asked for it): u becomes 4294967289
     u = narrow<unsigned>(d);        // OK: throws narrowing_error
 
 ##### Enforcement
 
 A good analyzer can detect all narrowing conversions. However, flagging all narrowing conversions will lead to a lot of false positives. Suggestions:
 
-* flag all floating-point to integer conversions (maybe only `float`->`char` and `double`->`int`. Here be dragons! we need data)
-* flag all `long`->`char` (I suspect `int`->`char` is very common. Here be dragons! we need data)
-* consider narrowing conversions for function arguments especially suspect
+* Flag all floating-point to integer conversions (maybe only `float`->`char` and `double`->`int`. Here be dragons! we need data)
+* Flag all `long`->`char` (I suspect `int`->`char` is very common. Here be dragons! we need data)
+* Consider narrowing conversions for function arguments especially suspect
```

Comment:
These are complete sentences so should end in a period as well as stating with a capital letter.

Reply:
> These are complete sentences so should end in a period as well as stating with a capital letter.

Done. And also added missing periods in ES.47.
For better traceability I added a second commit. I assume that someone is squashing them before fast forward merging.